tox.ini

```diff
@@ -7,7 +7,7 @@ passenv = TESTING_SQLALCHEMY_DATABASE_URI
 
 commands =
     python --version
-    py.test --pyargs ihatemoney.tests
+    py.test --pyargs ihatemoney.tests {posargs}
```

Comment:
`posargs` allows to pass arguments to pytest via tox, e.g. `tox -e py311 -- -v path/to/filetest.py`

Reply:
Excellent, I wondered recently why I couldn't select a particular test this way.